routers/transactions.controller.js

```diff
@@ -68,6 +55,42 @@ class TransactionController {
     }
   }
 
+  async getAnnualStats(req, res) {
+    try {
+      const { familyId } = req.user;
+      const { month, year } = req.params;
+      const transes = await transactionModel.getFamilyAnnualReport(
+        familyId,
+        month,
+        year,
+      );
+      return responseNormalizer(200, res, { transes });
+    } catch (e) {
+      errorHandler(req, res, e);
+    }
+  }
+
+  async familyAuthorization(req, res, next) {
+    try {
+      const user = await userModel.findById(req.user);
```

Comment:
better simply check that `req.user.familyId` is present. This check won't work right now

Reply:
const {familyId} = req.user;
if (!familyId)
Like this?